vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -103,10 +103,17 @@ The following is a list of specifications for the bundle descriptor and its comp
 |`healthCheckPath`|String|No||Endpoint for a health check|
 |`deploymentBaseName`|String|No||Used to define custom pod names|
 |`roles`|String[]|No||Exposed security roles|
-|`env`|[EnvironmentVariable[]](#environment-variables-specification)|No||Required environment variables|
+|`env`|EnvironmentVariable[]|No||Required environment variables|
 |`commands`|[Command[]](#command-specification)|No||Custom command(s) definitions|
 
-#### Microservices Sample Code
+### Environment Variables Specification
+|Name|Type|Required|Description|
+|:-|:-|:-|:------------------------|
+|`name`|String|Yes|Name of the environment variable to inject|
+|`value`|String|No|Value to give to the environment variable|
+|`valueFrom`|SecretKeyRef|No|Reference to the Secret from which to fetch the value|
```

Comment:
I think this is incorrect. The options are name, value, and secretKeyRef per https://developer.entando.com/v7.1/tutorials/devops/plugin-environment-variables.html#prerequisites. So line valueFrom should be secretKeyRef in the first column.